lib/driver-manager.js

```diff
@@ -244,6 +250,39 @@ export async function resizeBrowser( driver, screenSize ) {
 	}
 }
 
+export function getChromeWindowSize( screenSize ) {
+	let windowSize;
+	if ( typeof screenSize === 'string' ) {
+		switch ( screenSize.toLowerCase() ) {
+			case 'mobile':
+				windowSize = '--window-size=400,1000';
+				break;
+			case 'tablet':
+				windowSize = '--window-size=1024,1000';
+				break;
+			case 'desktop':
+				windowSize = '--window-size=1440,1000';
+				break;
+			case 'laptop':
+				windowSize = '--window-size=1400,790';
```

Comment:
This is no big deal but what devices are we capturing with this one? Dimensions seems a bit uncommon just when eyeballing. Was this aiming at smaller notebooks? 13", 12"? Maybe 1280 by 800?  

<img width="1680" alt="screen shot 2018-11-06 at 1 58 42 pm" src="https://user-images.githubusercontent.com/6520461/48087069-575b9d80-e1cc-11e8-9a7e-d698283a9eb9.png">

Reply:
I'm not sure. I just used the sizes that were already being used. Perhaps @alisterscott can answer that

Reply:
I think it was for visual diffs but we shouldn't worry any more since we don't have them